user_test.go

```diff
@@ -55,3 +55,21 @@ func TestUserService_Create(t *testing.T) {
 		t.Error("Expected user. User is nil")
 	}
 }
+
```

Comment:
Can you add tests for the other functions you added as well?  (e.g `Add`, `Remove`)

Reply:
Sure

Reply:
`Add`, `Remove` are APIs of `Group` so I wrote its tests in `group_test.go`.